source/optional_features/delayed_capture/index.html.md.erb

```diff
@@ -5,7 +5,9 @@ weight: 133
 
 # Delayed capture
 
-GOV.UK Pay supports the delayed capture of payments.
+You can control how long it takes for GOV.UK Pay to transfer ('capture') a payment from a user's bank account.
```

Comment:
i'm having second thoughts about this wording as it could imply that we can control the time it takes for a payment to be transferred from a user's account to the merchant's bank account (2 days - and we can't change that). you can control when a payment is removed from a user's account, after it has been authorised...?

Reply:
How about "... to **take** ('capture') a payment from a user's bank account"?

Reply:
I agree that 'transfer' sounds too much like we're talking about when it will arrive in the merchant's account.

Reply:
'take' works :)

Reply:
👍